docs/rdbms/naming-convention.md

```diff
@@ -0,0 +1,134 @@
+---
+id: rdbms-naming-convention
+title: RDBMS Naming Convention
+sidebar_label: Naming Convention
+---
+
+#### The following convention should be followed for naming RDBMS:
+
+* Use `snake_case` with lowercase while naming.
+* Use uppercase for SQL keywords and built-in functions.
+* Avoid use of dots, spaces, or dashes in database, schema, table, or column names.
+
+### Database
+
+Example `dvd_rental`.
+
+```sql
+--
+-- Good
+CREATE DATABASE dvd_rental;
+
+--
+-- Bad
+CREATE DATABASE [dvd rental];
+```
+
+### Schema
+
+Schema works as a namespace. Example `raw`, `ops`, `util`.
+
+```sql
+CREATE SCHEMA util;
+```
+
+### Table
+
+Example `actor`, `staff`, `category`, `film`, `film_category`, `film_actor`, `customer`, `sales`.
+
+```sql
+CREATE TABLE dbo.actor (
+  id INT UNSIGNED NOT NULL AUTO_INCREMENT,
+  first_name VARCHAR(45) NOT NULL,
+  last_name VARCHAR(45) NOT NULL,
+  created_at TIMESTAMP NOT NULL
+);
+```
+
+### View
+
+Example `actor_info`, `staff_list`, `film_list`, `customer_list`, `sales_by_category`.
+
+```sql
+CREATE VIEW report.staff_list
```

Comment:
`CREATE VIEW report.vw_staff_list` . Prefix with `vw_` can be useful to see quickly whether you are selecting from a view or table.